stream-chat-android-pushprovider-xiaomi/build.gradle

```diff
@@ -57,5 +57,5 @@ tasks.withType(org.jetbrains.kotlin.gradle.tasks.KotlinCompile).all {
 dependencies {
     implementation project(':stream-chat-android-client')
     implementation Dependencies.moshiKotlin
-    compileOnly files('libs/MiPush_SDK_Client_4_8_2.jar')
+    compileOnly files('libs/MiPush_SDK_Client_5_0_6-G_3rd.aar')
```

Comment:
It needs to be added as a local library yet, they don't provide any remote repository where download it :(
I am keeping the full name of the download artifact to know it is exactly the same artifact we downloaded